### PR TITLE
RBuilder, RDOMBuilder, RElementBuilder, StyledElementBuilder, StyledD…

### PR DESCRIPTION
**Pull request by Skolotsky**
RBuilder, RDOMBuilder, RElementBuilder, StyledElementBuilder, StyledDOMBuilder -> interfaces

### CODE REVIEW

File: kotlin-react-dom/src/main/kotlin/react/dom/RDOMBuilder.kt [hunk 1]
```diff
@@ -18,13 +18,76 @@ external interface DOMProps : WithClassName {
...
+
+    var SELECT.value: String
+        get() = get("value")
+        set(value) {
+            set("value", value)
+        }
+
+    val domProps: DOMProps
+
+//    init {
```
Comment (Leonya) [inline kotlin-react-dom/src/main/kotlin/react/dom/RDOMBuilder.kt:56]: Delete?

**Review by Leonya — CHANGES_REQUESTED** [review]

File: kotlin-react-dom/src/main/kotlin/react/dom/RDOMBuilder.kt (hunk 1 above)
Comment (Skolotsky) [reply kotlin-react-dom/src/main/kotlin/react/dom/RDOMBuilder.kt]: deleted

**Comment by Leonya** [comment]
@Skolotsky compilation is failing, see above.

**Review by Skolotsky — COMMENTED** [review]
> @Skolotsky compilation is failing, see above.

fixed import in redux module

**Pull request merged**